src/ucp/api/ucp_def.h

```diff
@@ -321,11 +321,11 @@ typedef void (*ucp_send_callback_t)(void *request, ucs_status_t status);
 
  /**
  * @ingroup UCP_COMM
- * @brief Completion callback for non-blocking sends ucp_tag_send_nbx call.
+ * @brief Completion callback for non-blocking sends.
  *
- * This callback routine is invoked whenever the @ref ucp_tag_send_nbx
- * "send operation" is completed. It is important to note that the call-back is
- * only invoked in a case when the operation cannot be completed in place.
+ * This callback routine is invoked whenever the @ref ucp_tag_send_nbx,
+ * @ref ucp_am_send_nbx, ucp_stream_send_nbx, ucp_put_nbx, ucp_get_nbx,
+ * ucp_atomic_op_nbx or any other "send operation" is completed.
```

Comment:
`"send operations" are completed`?

Reply:
no, because the callback completes a single operation

Reply:
sorry, my bad